lib/ansible/plugins/action/win_reboot.py

```diff
@@ -66,7 +69,7 @@ def perform_reboot(self):
             result1 = self._low_level_execute_command('shutdown /a', sudoable=self.DEFAULT_SUDOABLE)
 
             # Initiate reboot again
-            result2 = self._low_level_execute_command('shutdown /r /t %d' % pre_reboot_delay, sudoable=self.DEFAULT_SUDOABLE)
+            result2 = self._low_level_execute_command('shutdown /r /t {0}'.format(self.pre_reboot_delay), sudoable=self.DEFAULT_SUDOABLE)
```

Comment:
Should we be using `reboot_command` again. I think the only difference between the current and that is the latter has a comment added which is ok to use here.

Reply:
Seems reasonable. I wasn't sure if attempt number two needed a specifically different command for some reason. I'll use `reboot_command` here.